pages/common/mdp.md

```diff
@@ -0,0 +1,19 @@
+# mdp
+
+> A command-line, ncurses based tool to make presentations from markdown files.
+
+- Launch a presentation from a markdown file:
```

Comment:
Yeah I hoped ncurses would give that idea, but the description is now a bit technical so... trying something else...